### ISSUE TITLE
Double hyphen in hyphenation

### ISSUE THREAD

**Issue by garrettm30** [huertatipografica/Andada-Pro#29]
This was originally reported in issue #27, where it was a closed as a Chromium issue but with the offer to reopen if discovered elsewhere.

I can't myself repopen the issue, so I would like to report that I also am encountering the same issue (double hyphen in automatic hyphenation) in the layout application Affinity Publisher on Mac. This is the only font I can find that does this. If I change the text to, for example, Alegreya or Bitter, the automatic hyphenation uses a single hyphen as expected.

**Comment by BeckyDTP** [comment]
Manually adding U+00AD SOFT HYPHEN = `&#xAD` = `&#173;` = `&shy;` also displays a double hyphen.

Screenshot 1: Sigil with Preview (Chrome adopted by Qt).
![andada-pro-chrome-preview-in-Sigil](https://user-images.githubusercontent.com/22416021/213656861-6e33dd39-42bf-4739-ab34-9d07dd5c24b6.png)

Screenshot 2: Kindle Voyage (EPUB file converted to MOBI/AZW)
![andada-pro-kindle-voyager](https://user-images.githubusercontent.com/22416021/213656853-1a810f78-6a91-415e-985d-de4d97443a25.png)

**Comment by CaroGiovagnoli** [comment]
I've pushed a fix for this issue, please test it and feel free to close the issue if it is fixed.

**Comment by garrettm30** [comment]
> I've pushed a fix for this issue, please test it and feel free to close the issue if it is fixed.

I originally reported while using the first version of Affinity Publisher, and since then Publisher 2 has been released. In testing your changes, I discovered to my surprise that the previous version of your font does not manifest the problem Publisher 2.

In any case, by way of a baseline comparison, I first loaded the original Publisher with the previous version of the fonts to confirm that I can still reproduce the problem. Indeed, with the previous version of your fonts and Publisher 1, I confirmed the problem does still exist in that combination. Then I loaded up the latest version of your font and tested all weights in both Publisher 1 and 2. The soft hyphens now behave as expected, and I consider this issue fixed.

Many thanks to you for your attention to this detail, and even more, thank you for providing this wonderfully made font family. I am very glad to be able to use it now in our work.
